lib/SYCL/inSPIRation.cpp

```diff
@@ -41,6 +41,8 @@ using namespace llvm;
 /// Displayed with -stats
 STATISTIC(SYCLKernelProcessed, "Number of SYCL kernel functions processed");
 
+cl::opt<bool> ReqdWorkGroupSizeFPGA("reqd-workgroup-size-fpga", cl::desc("set reqd_work_group_size to be 1-1-1"));
```

Comment:
Would be better with a more modern syntax with `{ }` everywhere instead of `( )`

Reply:
It is not clear to me why there is `FPGA` or `-fpga` in the name.

Reply:
Should the option would be named `sycl-reqd-workgroup-size-1` instead?

Reply:
Would be better with a more modern syntax with { } everywhere instead of ( ).
Not sure about this. I checked *cl::opt<bool>* and *cl::desc* using in LLVM other passes are all with () instead of {}. Do you think keeping () is a good idea.


Should the option would be named *sycl-reqd-workgroup-size-1* instead?
Yes, I think naming is better.

Thanks! :-)